ecosystem/sep-0006.md

```diff
@@ -445,6 +461,7 @@ The wallet must use the response fields in the following way to complete the wit
 - `withdraw_memo`: (if specified) use this memo in the payment transaction to the anchor.
 - `withdraw_memo_type`: use this as the memo type.
 - `amount_in`: the amount expected in the Stellar payment.
+- `expires_at`: (if specified) if already expired, the wallet should not allow the user to proceed with the deposit/withraw action.
```

Comment:
💡  I think we can omit this from the change since this field is retrievable from the [`GET /quote`](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md#get-quote) endpoint and we don't need to duplicate information. The expiration also only refers to the delivery of funds to the anchor, vs. the entire transaction completing, so I think its confusing to list it on the transaction object in this SEP.

Reply:
Agreed! Removed it in https://github.com/stellar/stellar-protocol/pull/973/commits/169e4a1408ddafe6cd87a2008f4384e3a363403f.